src/libraries/Common/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -163,7 +151,7 @@ public override byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
                         tmp.CopyTo(destination);
                     }
 
-                    CryptographicOperations.ZeroMemory(tmp);
+                    CryptographicOperations.ZeroMemory(tmp.Slice(0, bytesWritten));
```

Comment:
Wass this just for clarity? `tmp` is sliced above as `tmp = tmp.Slice(0, bytesWritten);`

Reply:
Wellllll, it does mean that when destination is too small it doesn't clear tmp, but that's probably not actually desired.  I think I just made a poor edit when editing `stackalloc[0]` + conditonal `stackalloc[keySizeBytes]` to `stackalloc[512]`. :smile: